superset-frontend/src/profile/components/CreatedContent.tsx

```diff
@@ -53,7 +56,7 @@ class CreatedContent extends React.PureComponent<CreatedContentProps> {
     const search = [{ col: 'created_by', opr: 'created_by_me', value: 'me' }];
     const query = rison.encode({
       keys: ['none'],
-      columns: ['created_on_delta_humanized', 'dashboard_title', 'url'],
+      columns: ['changed_on_utc', 'dashboard_title', 'url'],
```

Comment:
Please add context on this change... doesn't seem related to the general translation theme of the PR.